beacon/blockchain/metrics.go

```diff
@@ -107,12 +107,3 @@ func (cm *chainMetrics) markOptimisticPayloadBuildFailure(
 		err.Error(),
 	)
 }
-
-// measureStateRootVerificationTime measures the time taken to verify the state
-// root of a block.
-// It records the duration from the provided start time to the current time.
-func (cm *chainMetrics) measureStateRootVerificationTime(start time.Time) {
```

Comment:
metric used for ProcessProposal, replaced by its FinalizeBlock companion. TODO: consider activating this under fork